inc/Study.php

```diff
@@ -488,6 +488,15 @@ public static function get_group_studies( $group_id = null ) {
 
 		$studies = groups_get_groupmeta( $group_id, '_sc_study', true );
 
+		// Loop through the studies and make sure they actually exist
+        if ( is_array( $studies ) && ! empty( $studies ) ) {
+            foreach( $studies as $key => $study ) {
+                if ( ! self::group_study_is_valid( $study ) ) {
+                    unset( $studies[ $key ] );
+                }
+            }
+        }
+
```

Comment:
@nickyoung87 let's add one more bit here that checks to see if the $studies array has changed, and if it has then update the group meta. I don't want to have a bunch of groups with orphaned studies.